src/browser/ui/__tests__/ReactMount-test.js

```diff
@@ -144,4 +151,25 @@ describe('ReactMount', function() {
     ReactMount.render(<div />, container);
     expect(console.warn.mock.calls.length).toBe(0);
   });
+
+  ;(WebComponents === undefined ? xit : it)
```

Comment:
No clue, it came with a copy-paste from ReactWebWorker-test, I suppose I was just being consistent :).